docs/tutorial/vscode.md

```diff
@@ -0,0 +1,26 @@
+# VSCode
+
+A lot of people on this project use VSCode as their coding environment, and there are a number of useful plugins available to make work more efficient:
+
+- c/c++ intellisense
+- clang-format
+- hexinspector (hover on numbers for float and other info)
+- numbermonger (hex to decimal and vice versa)
+- bracket pair colorizer 2
+- better mips support
+
+
+
+You will probably also find it helpful to watch Fig's video, although it was not meant to be an actual introduction. But it does show various useful features of VSCode, quite apart from the decompilation aspect.
+
+Useful stuff to know:
+
+- Ctrl + Alt + Up/Down gives multicursors across consecutive lines. If you want several cursors in a more diverse arrangement, middle clicking works, at least on Windows.
```

Comment:
For some reason, in Linux this shortcut is `Ctrl` + `Alt` + `Up`/`Down`

Reply:
VS Code's not very consistent across OSes, but is that actually different?

Reply:
Yes, it is different.
![image](https://user-images.githubusercontent.com/7416381/111852781-0bba7200-88f7-11eb-926d-287bd4eb9e93.png)

Also, the other shortcuts are the same as you described them.